examples/js/postprocessing/SSAOPass.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+/*
+ * Screen-space ambient occlusion pass.
+ *
+ * Has the following parameters
+ *  - radius
+ *  - onlyAO
+ *  - aoClamp
+ *  - lumInfluence
+ * 
+ * To output to screen set renderToScreens true
+ *
+ * @author alteredq / http://alteredqualia.com/
+ * @author tentone
+ */
+THREE.SSAOPass = function ( scene, camera, width, height ) {
+
+	if ( THREE.SSAOShader === undefined)
+	{
+		console.warn('THREE.SSAOPass depends on THREE.SSAOShader');
```

Comment:
Just some minor code style feedback: There should be spaces before and after the brackets like this:
```javascript
console.warn( 'THREE.SSAOPass depends on THREE.SSAOShader' );
```